instructions/docs/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+## Background
+
+In this section, you will prepare your AWS Environment to work with the Busy Engineer's Document Bucket. You will:
```

Comment:
What is the aim of learning these things? e.g. "Gain familiarity with the core concepts of encryption using a key provider, and become familiar with encrypting and decrypting across multiple regions"

Reply:
Good callout. Will add this expansion.